### PR TITLE
Add fabric

### PR DESCRIPTION
**Pull request by advaypal**
Adds fabric to the project

### CODE REVIEW

File: MIDIchlorians/MIDIchlorians/Info.plist [hunk 1]
```diff
@@ -14,28 +14,42 @@
...
-	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
-	<key>LSRequiresIPhoneOS</key>
-	<true/>
+	<key>Fabric</key>
+	<dict>
+		<key>APIKey</key>
+		<string>56683ad8cc8f1cf4ed01f5000ee9e316e315fc62</string>
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians/Info.plist:35]: api key

**Review by ngzhian — CHANGES_REQUESTED** [review]

File: MIDIchlorians/MIDIchlorians/Info.plist (hunk 1 above)
Comment (advaypal) [reply MIDIchlorians/MIDIchlorians/Info.plist:35]: Has to be there.. Don't know how to get around this

Comment (ngzhian) [reply MIDIchlorians/MIDIchlorians/Info.plist:35]: maybe we can try this? https://twittercommunity.com/t/how-to-prevent-publication-of-fabric-api-key-in-open-source-xcode-projects/69250

Comment (ngzhian) [reply MIDIchlorians/MIDIchlorians/Info.plist:35]: or this: https://twittercommunity.com/t/should-apikey-be-kept-secret/52644/6

Comment (ngzhian) [reply MIDIchlorians/MIDIchlorians/Info.plist:35]: well, "The Fabric/Crashlytics API key is a write-only token designed to be exposed. However, it's still good practice to not publicly publish it.", so I say don't care and push this, gonna change this to approved

**Review by ngzhian — APPROVED** [review]

**Pull request merged**